src/main/java/net/atos/zac/app/taken/TakenRESTService.java

```diff
@@ -232,7 +232,8 @@ public RESTTaak completeTaak(final RESTTaak restTaak) {
         }
 
         createDocuments(restTaak);
-        ondertekenEnkelvoudigInformatieObjecten(restTaak.taakdata);
+        final Zaak taakZaak = zrcClientService.readZaak(restTaak.zaakUuid);
```

Comment:
Noem de taakZaak variabele maar gewoon zaak. In de context van deze code is het is duidelijk dat het de zaak is waar de taak bij hoort,

Reply:
Opgelost in opvolgende commit